kinto/views/groups.py

```diff
@@ -1,6 +1,8 @@
 import colander
```

Comment:
Changes in this file aren't related to the PR, are they? You took the opportunity to change the group deletion to using the subscriber too?